### ISSUE TITLE
cb-never-called-1.0.1.tgz - suspected in a malware scan

### ISSUE THREAD

**Issue by TheILPlace** [angular/angular-cli#4161]
> Please provide us with the following information:
> ---------------------------------------------------------------

### OS?
> Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)

win10

### Versions.
> Please run `ng --version`. If there's nothing outputted, please run in a Terminal: `node --version` and paste the result here:

cli ver 24
### Repro steps.
> Was this an app that wasn't created using the CLI? What change did you do on your code? etc.

### The log given by the failure.
> Normally this include a stack trace and some more information.

### Mention any other details that might be useful.

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

c:\projects\first-app\first-app\node_modules\tar\test\cb-never-called-1.0.1.tgz
..roaming\npm\npm\node_modules\angular-cli\node_modules\tar\test\cb-never-called-1.0.1.tgz

those files are failing a malware scan.
what is this file ? and can it be ignored ..

btw, trying to open it in winrar, fails with 
   \node_modules\tar\test\cb-never-called-1.0.1.tgz: Unexpected end of archive
    \node_modules\tar\test\cb-never-called-1.0.1.tgz: The archive is corrupt

thanks

**Comment by clydin** [comment]
It's a test file for the package `tar` which is a transitive dependency.  See here for a request to not publish the test files: https://github.com/npm/node-tar/pull/72

**Comment by TheILPlace** [comment]
and in plain english ? can i remove it from my cli and angular project folder ?

**Comment by clydin** [comment]
Most likely.  However, a definitive answer (and any potential overall solution) should really come from the maintainer of the package in question. (https://github.com/npm/node-tar)
